native/cocos/base/Log.cpp

```diff
@@ -204,6 +206,26 @@ void Log::logMessage(LogType type, LogLevel level, const char *formats, ...) {
         default:
             HILOG_DEBUG(LOG_APP, typeStr, buff);
     }
+#elif (CC_PLATFORM == CC_PLATFORM_OPENHARMONY)
+    ::LogLevel ohosLoglevel = ::LogLevel::LOG_DEBUG;
+    switch (level) {
+        case LogLevel::LEVEL_DEBUG:
+        ohosLoglevel = ::LogLevel::LOG_DEBUG;
+        break;
+        case LogLevel::INFO:
+        ohosLoglevel = ::LogLevel::LOG_INFO;
+        break;
+        case LogLevel::WARN:
+        ohosLoglevel = ::LogLevel::LOG_WARN;
+        break;
+        case LogLevel::ERR:
+        ohosLoglevel = ::LogLevel::LOG_ERROR;
+        break;
+        case LogLevel::FATAL:
+        ohosLoglevel = ::LogLevel::LOG_FATAL;
+        break;
+    }
+    OH_LOG_Print(LOG_APP, ohosLoglevel, LOG_DOMAIN, "HMG_LOG", "%{public}s", buff);
```

Comment:
What does `%{public}s` stand for?

Reply:
The requirement of openharmony to print the log, otherwise it will not be printed